tests/common.go

```diff
@@ -9198,6 +9191,38 @@ func GetClusterProvisionStatus() ([]ProvisionStatus, error) {
 			}
 		}
 	}
+	log.Infof("Cluster provision status [%v]", clusterProvision)
+	return clusterProvision, nil
+}
+
+// GetClusterProvisionStatus  returns details of cluster provision status
+func GetClusterProvisionStatus() ([]ProvisionStatus, error) {
+	// Using Node which is up and running
+	var selectedNode []node.Node
+	for _, eachNode := range node.GetStorageDriverNodes() {
+		status, err := IsPxRunningOnNode(&eachNode)
+		if err != nil {
+			log.InfoD("Px is not running on the Node.. searching for other node")
+			continue
+		}
+		if status {
+			selectedNode = append(selectedNode, eachNode)
```

Comment:
can we select one node and break this loop? any particular reason you want all nodes and then select random node?

Reply:
oh !! I thought I was breaking it , let me reupdate it similar to above ones

Reply:
the intension here is to pick up random node , for specific node we already have a function GetClusterProvisionStatusOnSpecificNode which can be used